src/BroadcasttClient.php

```diff
@@ -324,7 +324,7 @@ public function buildAuthQueryString($requestMethod, $requestPath, $queryParams
      *
      * @return string The imploded array
      */
-    public static function httpBuildQuery($array)
+    public static function httpBuildQuery($array): string
```

Comment:
The PHPDoc is incorrect here `$array` must be an `array` according to the implementation. It might be a typo and it wanted to be `string[]` in the doc block.